package.json

```diff
@@ -47,7 +47,7 @@
     "tap": "^10.0.1"
   },
   "engines": {
-    "node": ">=0.10",
+    "node": ">=8",
     "npm": ">=2"
```

Comment:
I'm all for it, if we have a reason. E.g. for security we could enforce sha512 hashes when publishing.

But this package.json entry doesn't really do much, so we'd have to write some additional code first. I'd do all of this in a separate step.